Jenkinsfile

```diff
@@ -7,13 +7,13 @@ def oss = ["linux"]
 def builds = [:]
 for (def os in oss) {
   for (def jdk in jdks) {
-    builds[os+"_"+jdk] = getFullBuild( jdk, os )
+    builds[os+"_"+jdk] = getFullBuild( jdk, os, mainJdk)
```

Comment:
Can this just pass a boolean variable on the 3rd param?
Something like "isMainBuild"?
Then we can drop the "def isMainBuild" method too.